cypress/utils/utils.ts

```diff
@@ -456,6 +459,53 @@ export function getTableColumnData(columnName: string): Array<string> {
         });
     return itemList;
 }
+export function verifyDateSortAsc(listToVerify: Array<string>, unsortedList: Array<string>): void {
+    cy.wrap(listToVerify).then((capturedList) => {
+        let sortedList = [...unsortedList]
+            .map((dateStr) => {
+                // Manually parse the date
+                const [month, day, year] = dateStr.split("/");
+                return new Date(parseInt(year), parseInt(month) - 1, parseInt(day));
+            })
+            .sort((a, b) => a.getTime() - b.getTime()) // sort the dates
+            .map((date) => {
+                // Convert the date back to "MM/DD/YYYY" format.
+                let formattedDate =
+                    (date.getMonth() + 1).toString().padStart(2, "0") +
+                    "/" +
+                    date.getDate().toString().padStart(2, "0") +
+                    "/" +
+                    date.getFullYear();
+                return formattedDate;
+            });
```

Comment:
The date format may vary depending on the execution environment, try to refactor this function to be agnostic of the date format. Ask any questions you need :smiley: same for the `verifyDateSorDesc` method